lib/msal-common/src/cache/CacheManager.ts

```diff
@@ -233,11 +233,6 @@ export abstract class CacheManager implements ICacheManager {
      */
     abstract getTokenKeys(): TokenKeys;
 
-    /**
-     * Function which clears cache.
-     */
-    abstract clear(): Promise<void>;
```

Comment:
Would it make more sense to change the common abstract code to
`abstract clear(): Promise<void> | void;`
instead of removing it? Or make a second abstract
`abstract clear(): void;`
?

Reply:
These would make it unclear whether clear is supposed to be ambiguous or ambiguous. Either way, there isn't an instance where CacheManager needs to call clear, so it should be fine.